scripts/just.config.ts

```diff
@@ -1,29 +1,31 @@
-import { task, series, parallel, condition, option, argv, addResolvePath, resolveCwd } from 'just-scripts';
-
-import path from 'path';
-import fs from 'fs';
-
-import { clean } from './tasks/clean';
-import { copy } from './tasks/copy';
-import { jest, jestWatch } from './tasks/jest';
-import { sass } from './tasks/sass';
-import { ts } from './tasks/ts';
-import { tslint } from './tasks/tslint';
-import { webpack, webpackDevServer } from './tasks/webpack';
-import { verifyApiExtractor, updateApiExtractor } from './tasks/api-extractor';
-import lintImports from './tasks/lint-imports';
-import prettier from './tasks/prettier';
-import bundleSizeCollect from './tasks/bundle-size-collect';
-import checkForModifiedFiles from './tasks/check-for-modified-files';
-import generateVersionFiles from './tasks/generate-version-files';
-import generatePackageManifestTask from './tasks/generate-package-manifest';
-import { postprocessTask } from './tasks/postprocess';
-import { postprocessAmdTask } from './tasks/postprocess-amd';
-import { postprocessCommonjsTask } from './tasks/postprocess-commonjs';
-import { startStorybookTask, buildStorybookTask } from './tasks/storybookTask';
+// @ts-check
```

Comment:
Is there a reason this is turned back from typescript syntax back to javascript?

Reply:
The change to TS syntax was actually just in a particular commit of this PR, not in master. It broke some other stuff, so I turned it back for now for simplicity.